sklearn/datasets/descr/linnerud.rst

```diff
@@ -9,7 +9,8 @@ Linnerrud dataset
     :Number of Attributes: 3
     :Missing Attribute Values: None
 
-The Linnerud dataset constains two small dataset:
+The Linnerud dataset consists of three excercise (data) and three physiological
+(target) variables collected from twenty middle-aged men in a fitness club:
 
```

Comment:
Maybe make it even more explicit by stating that this is a multi-output regression dataset (which is not that common in scikit-learn).